scripts/pre_commit_linter.py

```diff
@@ -80,34 +78,45 @@ def _is_javascript_file(filename):
     return filename.endswith('.js')
 
 
-def _lint_js_files(node_path, jscs_path, autofix, config_jscsrc):
+def _lint_js_files(node_path, jscs_path, config_jscsrc, input_dir):
     """Prints a list of lint errors in changed JavaScript files.
 
     Args:
     - node_path: str. Path to the node binary.
     - jscs_path: str. Path to the JSCS binary.
-    - autofix: bool. Whether to automatically fix errors.
+    - input_dir: str. Path to the folder to be linted(Optional).
     """
     changed_filenames = _get_changed_filenames()
     changed_js_filenames = filter(_is_javascript_file, changed_filenames)
     num_js_files = len(changed_js_filenames)
+    is_directory = False
+    if input_dir:
```

Comment:
Better to do this in the main method, and print an error (and exit) if the directory is invalid.